definitions/divorce/json/CaseRoles.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "[RESPSOLICITOR]",
```

Comment:
Case roles should not yet be in prod

Reply:
Same with PETSOLICITOR below

Reply:
We can add the roles here as long as we don't use them anywhere...

Reply:
Yeah as far as I know, this will only trigger in Notice of Acting which is behind the superuser role?